x-pack/legacy/plugins/apm/public/components/app/ServiceNodeMetrics/index.tsx

```diff
@@ -0,0 +1,120 @@
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiTitle,
+  EuiHorizontalRule,
+  EuiFlexGrid,
+  EuiPanel,
+  EuiSpacer,
+  EuiStat
+} from '@elastic/eui';
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import styled from 'styled-components';
+import { ApmHeader } from '../../shared/ApmHeader';
+import { useUrlParams } from '../../../hooks/useUrlParams';
+import { useAgentName } from '../../../hooks/useAgentName';
+import { useServiceMetricCharts } from '../../../hooks/useServiceMetricCharts';
+import { ChartsSyncContextProvider } from '../../../context/ChartsSyncContext';
+import { MetricsChart } from '../../shared/charts/MetricsChart';
+import { useFetcher, FETCH_STATUS } from '../../../hooks/useFetcher';
+import { callApmApi } from '../../../services/rest/callApmApi';
+import { truncate, px, unit } from '../../../style/variables';
+
+const INITIAL_DATA = {
+  host: '',
+  containerId: ''
+};
+
+const Truncate = styled.span`
+  display: block;
+  ${truncate(px(unit * 12))}
+`;
+
+export function ServiceNodeMetrics() {
+  const { urlParams } = useUrlParams();
+  const { serviceName, serviceNodeName } = urlParams;
+
+  const { agentName } = useAgentName();
+  const { data } = useServiceMetricCharts(urlParams, agentName);
+  const { start, end } = urlParams;
+
+  const {
+    data: { host, containerId } = INITIAL_DATA,
+    status
+  } = useFetcher(() => {
+    if (serviceName && serviceNodeName) {
+      return callApmApi({
+        pathname:
+          '/api/apm/services/{serviceName}/node/{serviceNodeName}/metadata',
+        params: {
+          path: { serviceName, serviceNodeName }
+        }
+      });
+    }
+  }, [serviceName, serviceNodeName]);
+
+  const isLoading =
+    status === FETCH_STATUS.LOADING || status === FETCH_STATUS.PENDING;
+
+  return (
+    <div>
+      <ApmHeader>
+        <EuiFlexGroup alignItems="center">
+          <EuiFlexItem grow={false}>
+            <EuiTitle size="l">
+              <h1>{serviceName}</h1>
+            </EuiTitle>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </ApmHeader>
+      <EuiHorizontalRule margin="m" />
+      <EuiFlexGroup gutterSize="xl">
+        <EuiFlexItem grow={false}>
+          <EuiStat
+            titleSize="s"
+            isLoading={isLoading}
+            description={i18n.translate('xpack.apm.serviceNodeMetrics.host', {
+              defaultMessage: 'Host'
+            })}
+            title={<Truncate>{host}</Truncate>}
+          ></EuiStat>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiStat
+            titleSize="s"
+            isLoading={isLoading}
+            description={i18n.translate(
+              'xpack.apm.serviceNodeMetrics.containerId',
+              {
+                defaultMessage: 'Container ID'
+              }
+            )}
+            title={<Truncate>{containerId}</Truncate>}
```

Comment:
@dgieselaar Adding `Truncate` doesn't display the full value in a tooltip? I'd expect to be able to see the full ID, we just intentionally hid it from being written out in the detail header.

Reply:
Added the tooltip 👍